### PR TITLE
Simple-servlet based web interface has been added

### PR DESCRIPTION
**Pull request by cosmiclavaflow**

### CODE REVIEW

File: src/main/java/com/academy/persistence/dao/GenericDao.java [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
 
+import java.util.List;
+
 public interface GenericDao<T> {
 
     void create(T entity);
 
+    List<T> findAll(Class<T> type);
+
     T findById(Long id);
```
Comment (Anderwerty) [inline src/main/java/com/academy/persistence/dao/GenericDao.java]: What do you think about using Optional here?
T findById(Long id); -> Optional<T> findById(Long id);

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE web-app PUBLIC
+        "-//Sun Microsystems, Inc.//DTD Web Application 2.3//EN"
+        "http://java.sun.com/dtd/web-app_2_3.dtd" >
+
+<web-app>
+    <display-name>Archetype Created Web Application</display-name>
```
Comment (Anderwerty) [inline src/main/webapp/WEB-INF/web.xml]: Please use own application name.

File: src/main/webapp/index.jsp [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<%@ page isELIgnored="false" language="java" contentType="text/html; charset=ISO-8859-1"
```
Comment (Anderwerty) [inline src/main/webapp/index.jsp]: Why not utf-8?

File: src/main/java/com/academy/controller/HomeController.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+@WebServlet("/list")
+public class HomeController extends HttpServlet {
+
+    private GenericDao developerDao = new DeveloperDao();
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        Map<String, String[]> parameterMap = request.getParameterMap();
+
+        List<Developer> developers = (List<Developer>) developerDao.findAll(Developer.class);
```
Comment (Anderwerty) [inline src/main/java/com/academy/controller/HomeController.java]: Why do we need cast here?

Comment (cosmiclavaflow) [reply src/main/java/com/academy/controller/HomeController.java]: I mindlessly replaced `T findById()` from test fragment to `List<T>findAll()` )

**Pull request merged**